specification/e-referrals-service-api.yaml

```diff
@@ -1038,7 +1041,7 @@ paths:
         | `ReferralRequest` that was converted from an Advice and Guidance Request | `ubrn`=`000000070004`   |
         | `ReferralRequest` with related `ReferralRequest`                         | `ubrn`=`000000070005`   |
 
-      summary: A005 - Retrieve Referral Request
+      summary: A005 - Retrieve referral request
```

Comment:
Has it been decided that we want to go with lower case as opposed to title case for all endpoint names?

Reply:
I assumed we wanted t follow what the majority was using = sentence case. From https://nhsd-jira.digital.nhs.uk/browse/ERSSUP-60878: 

> Endpoints A005 and A031 onwards are in title case (e.g. A032 - Change Shortlist and Send for Triage), whilst the majority of endpoints are in sentence case (e.g. A027 - Convert advice and guidance request to referral).